library/src/androidTest/java/com/google/maps/android/data/kml/KmlTestUtil.java

```diff
@@ -0,0 +1,37 @@
+package com.google.maps.android.data.kml;
+
+import org.xmlpull.v1.XmlPullParser;
+import org.xmlpull.v1.XmlPullParserException;
+import org.xmlpull.v1.XmlPullParserFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+import androidx.annotation.IdRes;
+import androidx.test.platform.app.InstrumentationRegistry;
+
+/**
+ * Shared utilities for KML tests
+ */
+public class KmlTestUtil {
+
+    /**
+     * Returns an XmlPullParser for the given KML file
+     * @param res the resource ID for a KML file
+     * @return an XmlPullParser for the given KML file
+     * @throws XmlPullParserException, IOException
+     */
+    static XmlPullParser createParser(@IdRes int res) throws XmlPullParserException, IOException {
+        InputStream stream =
+                InstrumentationRegistry.getInstrumentation()
+                        .getTargetContext()
+                        .getResources()
+                        .openRawResource(res);
+        XmlPullParserFactory factory = XmlPullParserFactory.newInstance();
+        factory.setNamespaceAware(true);
+        XmlPullParser parser = factory.newPullParser();
+        parser.setInput(stream, null);
+        parser.next();
```

Comment:
Why does `next` have to be invoked here?

Reply:
Not sure why it was originally included (this was a direct refactor from another existing test class). Looks like it's not needed though, as I just removed it and the tests all pass without it (and `KmlLayer.createXmlParser()` also doesn't include it). Just pushed https://github.com/googlemaps/android-maps-utils/pull/614/commits/741667f3b53c481348574a76eeed94ce96a79eae with this change, and I'll merge after the CI passes.